server/core/terraform/async_client.go

```diff
@@ -8,13 +8,22 @@ import (
 
 	"github.com/hashicorp/go-version"
 	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/core/terraform/helpers"
 	"github.com/runatlantis/atlantis/server/events/command"
 	"github.com/runatlantis/atlantis/server/jobs"
 )
 
 // Setting the buffer size to 10mb
 const BufioScannerBufferSize = 10 * 1024 * 1024
 
+//go:generate pegomock generate -m --use-experimental-model-gen --package mocks -o mocks/mock_terraform_client_async.go ClientAsync
+
+type ClientAsync interface {
```

Comment:
Did you duplicate (+ mock) the`AsyncTFExec` interface to prevent more pkg imports?

Reply:
yep, otherwise it is creating cyclical dependency :(